docs/integration_resources/software/houdini.md

```diff
@@ -4,9 +4,12 @@
 
 Shift provides a `hshift` utility script to properly bind the Shift UI to Houdini. In order to run and open Shift in Houdini, execute the following code block inside the Houdini *Python Script Editor*:
 
-<pre><code style="white-space: pre; margin: 20px 0; padding: 10px; box-sizing: border-box;">import sys
-sys.path.append("&ltpath_to_the_shift_installation_folder&gt")
-sys.path.append("&ltpath_to_the_shift_installation_folder&gt/shift/thirdparty/python/Lib/site-packages")
+<pre><code style="white-space: pre; margin: 20px 0; padding: 10px; box-sizing: border-box;">try:
+    import shift
+exception:
+    import sys
+    sys.path.append("<path_to_the_shift_installation_folder>")
+    sys.path.append("<path_to_the_shift_installation_folder>/shift/thirdparty/python/Lib/site-packages")

```

Comment:
This section does not display nicely
![image](https://github.com/user-attachments/assets/3f60d782-eed1-48ae-9d3f-41bab1995e72)
May be related to the < > or other thing

Reply:
Done